examples/on-conflict-clause/on_conflict_clause.bal

```diff
@@ -7,32 +7,53 @@ type Student record {|
 |};
 
 public function main() {
-    error onConflictError = error("Key Conflict", message = "record with same key exists.");
-
     table<Student> students = table [
-            {id: 1, name: "John", score: 100},
-            {id: 2, name: "Jane", score: 150},
-            {id: 1, name: "John", score: 200}
+            {id: 1, name: "Admin", score: 100},
```

Comment:
Why would there be an admin in students? We need to rethink the entire example if we are changing it like this.

Reply:
Changed the record name to `Employee`